src/utils/focusTimeout.js

```diff
@@ -1,16 +1,27 @@
-let outOfFocusStart;
+let outOfFocusStart = null; // enforce null otherwise it could be undefined and the callback wouldn't execute
 let timeOutOfFocus = 0;
 let suspendedTimeouts = [];
 
-document.addEventListener('visibilitychange', () => {
+function trackTimeOutOfFocus() {
   if (document.hidden) {
     outOfFocusStart = Date.now()
   } else {
-    timeOutOfFocus += Date.now() - outOfFocusStart
-    suspendedTimeouts.forEach(({ callback, startTime, setTimerId }) => setTimerId(setFocusTimeout(callback, timeOutOfFocus - startTime)()))
+    timeOutOfFocus += Date.now() - (outOfFocusStart); // when the page is loaded in hidden state outOfFocusStart is undefined, which results in timeoutOffset being NaN
     outOfFocusStart = null;
```

Comment:
You had a comment here calling this to be set before looping on suspendedTimeouts - I removed it as I don't think the order matters, it's checked from within a `setTimeout` callback.

Reply:
yes you're right, it shouldn't matter.